src/IPBlockIterator.php

```diff
@@ -33,7 +35,7 @@ class IPBlockIterator implements \Iterator, \Countable
     protected $first_block = null;
 
     /**
-     * @var int
+     * @var \GMP
```

Comment:
If this is a GMP instance, does it make sense to initialize it to "0"?

Reply:
Agreed